docs/information/configuration.md

```diff
@@ -160,6 +160,11 @@ experimental:
   # attribute: topic 'zigbee2mqtt/my_bulb/state' payload 'ON"
   # attribute_and_json: both json and attribute (see above)
   output: 'json'
+  # Optional: Transmit power setting (default: 5 for CC2652R and RB). 
+  # This was specifically added for devices (e.g. CC2652P and the CC1352P-2) 
+  # that have an inbuilt power amp with up to 20 dBm for longer range. 
+  # If you're using the CC253X or the CC2652R this doesn't do anything. 
```

Comment:
> CC2652R this doesn't do anything. 

This is not true, if you set `transmit_power: 0` it will be set to 0.

Also the value is a number not a string (`transmit_power: '5'` -> `transmit_power: 5`)